src/ar/arabshentai/src/eu/kanade/tachiyomi/extension/ar/arabshentai/ArabsHentai.kt

```diff
@@ -26,7 +26,7 @@ class ArabsHentai : ParsedHttpSource() {
 
     override val lang = "ar"
 
-    private val dateFormat = SimpleDateFormat("d MMM\u060c yyy", Locale("ar"))
+    private val dateFormat = SimpleDateFormat("d MMM\u060c yyyy", Locale("ar"))
```

Comment:
This change has no functional change. The format is also wrong. Currently they seem to use ISO8601.